Yosemite/Yosemite/Stores/OrderStore.swift

```diff
@@ -465,10 +486,24 @@ extension OrderStore {
 // MARK: - Storage: Search Results
 //
 private extension OrderStore {
+    /// Updates the specified ReadOnly Order Entity.
+    ///
+    /// - Returns: The updated order, prior to performing the update operation.
+    ///
+    @discardableResult
+    func updateStoredOrder(readOnlyOrder: Networking.Order) -> Networking.Order {
+        guard let storageOrder = storageManager.viewStorage.loadOrder(siteID: readOnlyOrder.siteID, orderID: readOnlyOrder.orderID) else {
+            return readOnlyOrder
+        }
+
+        let oldReadOnlyOrder = storageOrder.toReadOnly()
+        upsertStoredOrders(readOnlyOrders: [readOnlyOrder], in: storageManager.viewStorage)
+        return oldReadOnlyOrder
+    }
 
     /// Upserts the Orders, and associates them to the SearchResults Entity (in Background)
     ///
-    private func upsertSearchResultsInBackground(keyword: String, readOnlyOrders: [Networking.Order], onCompletion: @escaping () -> Void) {
```

Comment:
As this method already belongs to a private extension, this `private` keyword is redundant 🗑